### ISSUE TITLE
Feat: subscribe to newsletter is not working fine additionally it is not storing emails that are subscribed to newsletter.

### ISSUE THREAD

**Issue by VinayLodhi1712** [anuragverma108/SwapReads#4133]
### Describe the feature

There are problems in subscribe to newsletter component -
1. not storing emails that already subscribed.
2. a user can subscribed many times it does not give any response it it is already subscribed.
3. The confirmation mail should be more good looking.

### Add ScreenShots

![subsc](https://github.com/user-attachments/assets/51941f80-d7df-44ef-a2ef-491a471df5ea)
![fith](https://github.com/user-attachments/assets/e4c9b73f-26cf-4a92-a41b-48999a2efcf1)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by abhi03ruchi** [comment]
are u working on this issue ?

**Comment by VinayLodhi1712** [comment]
yes, but the program admin does not reply on this issue and also this issue is of level 3 or atleast level 2 so i am currently waiting. If admin assign this issue than i will work on it currently only github bot has commented on it.

**Comment by VinayLodhi1712** [comment]
> are u working on this issue ?

can you assign this issue so i will start working on it.

**Comment by abhi03ruchi** [comment]
in ur project , github bot by default assigns these labels .

work on the issue , and raise pr .

during pr review , depending upon ur work , we will assign u the level .
